models/README.md

```diff
@@ -0,0 +1,23 @@
+# Models
+
+There are **four** main models in this repository
+
+1. Lexical Model - Uses lexical features like POS tags, Named entities etc., followed by a Conditional Random Field model
+2. Lexical Plus Model - In addition to the lexical features, we engineered more optimization specific features followed by a Conditional Random Field model
+3. Semantic Model - Semantic model is a RoBERTa-based deep learning model fine-tuned on our training data. You find the model [here](https://huggingface.co/skadio/ner4opt-roberta-v1) 
+4. (Our best) **Hybrid Model** - This model combines the above feature engineering and feature learning models to get best results on our current dataset. 
```

Comment:
to be explicit, can we add "combines above feature engineering (#2) and "semantic" feature learning (#3) models to get the best performance." You can add a sentence like, see our paper (cite) for more details on our Hybrid Solution. 

You can drop the "on our current dataset". The data is not ours.

Reply:
But reading the below.. it sounds like we are not using #3 in #4.. we are using v2? and v2 and v1 are basically same, except some randomness??

Reply:
I edited to make it more explicit like you suggested. We are not using 3 in 4, for some reason using 3 in 4 gives slightly poorer results even though the model architecture is similar. For the same reasons of uncertainties/randomness, I saved the models we built and submitted to Ner4opt leaderboard too. Regarding citing the paper, do you have any suggestions on where we should upload our paper? Its not publicly available yet right?